tests/NHSD.BuyingCatalogue.Organisations.Api.IntegrationTests/Features/Healthchecks.3538.feature

```diff
@@ -5,12 +5,11 @@
 
 @5648
 Scenario: 1. Database Server is up
-
-	When the dependency health-check endpoint is hit for OAPI
+	When the dependency health-check endpoint is hit
```

Comment:
Again the number is different to the 3538, should this match?

Reply:
Same reason as before, will see what you think and then resolve / make changes.